/bin/bash: /data/users2/bthapaliya/anaconda-main/anaconda3/lib/libtinfo.so.6: no version information available (required by /bin/bash)
arctrdgn002
python: can't open file '/data/users2/bthapaliya/DistributedFLExperiments/DistributedFL/fedml_experiments/standalone/dpsgd/main_ditto.py': [Errno 2] No such file or directory
